<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>FourthWindow</class>
	<widget class="QMainWindow" name="FourthWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>800</width>
				<height>600</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<widget class="QGroupBox" name="groupBox">
				<property name="geometry">
					<rect>
						<x>130</x>
						<y>80</y>
						<width>551</width>
						<height>351</height>
					</rect>
				</property>
				<property name="title">
					<string>Send mail</string>
				</property>
				<widget class="QPushButton" name="pushButton_send">
					<property name="geometry">
						<rect>
							<x>160</x>
							<y>220</y>
							<width>221</width>
							<height>31</height>
						</rect>
					</property>
					<property name="text">
						<string>Send</string>
					</property>
				</widget>
				<widget class="QWidget" name="layoutWidget">
					<property name="geometry">
						<rect>
							<x>70</x>
							<y>88</y>
							<width>421</width>
							<height>111</height>
						</rect>
					</property>
					<layout class="QVBoxLayout" name="verticalLayout">
						<item>
							<layout class="QHBoxLayout" name="horizontalLayout">
								<item>
									<widget class="QLabel" name="label_to">
										<property name="text">
											<string>To</string>
										</property>
									</widget>
								</item>
								<item>
									<widget class="QLineEdit" name="lineEdit_to"/>
								</item>
								<item>
									<widget class="QLabel" name="label_title">
										<property name="text">
											<string>Title</string>
										</property>
									</widget>
								</item>
								<item>
									<widget class="QLineEdit" name="lineEdit_title"/>
								</item>
							</layout>
						</item>
						<item>
							<layout class="QHBoxLayout" name="horizontalLayout_2">
								<item>
									<widget class="QLabel" name="label_context">
										<property name="text">
											<string>Context</string>
										</property>
									</widget>
								</item>
								<item>
									<widget class="QLineEdit" name="lineEdit_context"/>
								</item>
							</layout>
						</item>
					</layout>
				</widget>
			</widget>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>800</width>
					<height>25</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
</ui>
